lib.ftd

```diff
@@ -5904,6 +5906,133 @@ color: $inherited.colors.background.step-1
 
 
 
+
+-- component plan-selected:
+common.benefits-record list benefits: $common.benefits
+caption title: Plan Selected
+string plan: Business+
+string plan-select: Annually Billed
+integer amount: 239
+children wrap:
+
+-- ftd.column:
+width.fixed.px: 720
+
+-- ftd.text: $plan-selected.title
+role: $fonts.font-18-24-600
+color: $inherited.colors.text
+margin-bottom.px: 8
+
+-- ftd.column:
+width: fill-container
+height.fixed.px: 457
+background.solid: $inherited.colors.background.step-1
+border-radius.px: 2
+border-width.px: 1
+border-color: $inherited.colors.custom.two
+padding-vertical.px: 20
+padding-horizontal.px: 20
+color: $fonts.005FBB
+
+-- ftd.row:
+background.solid: $fonts.c3d9ee
+padding-horizontal.px: 20
+padding-vertical.px: 20
+width: fill-container
+
+-- ftd.column:
+spacing.fixed.px: 16
+width.fixed.percent: 60
+
+-- ftd.row:
+spacing.fixed.px: 6
+
+-- ftd.text: $plan-selected.plan
+role: $fonts.font-16-21-700
+
+-- ftd.row:
+width.fixed.px: 4
+height.fixed.px: 4
+border-radius.px: 100
+background.solid: $fonts.005FBB
+align-self: center
+
+-- end: ftd.row
+
+-- ftd.text: $plan-selected.plan-select
+role: $fonts.font-16-21-700
+
+-- end: ftd.row
+
+-- ftd.row:
+spacing.fixed.px: 8
+role: $fonts.font-16-98-700
+
+-- ftd.text: \$
```

Comment:
```suggestion
-- ftd.text: <it should come from string property>
```
this seems to be currency and currency can be changed as per country code.

Reply:
$ is  for currency but we access each component property with $component.property-name. if we define a string property whose value is $ , it try to find variable which throws error